src/elarm_alarmlist.erl

```diff
@@ -70,26 +72,26 @@ repeat_alarm(_Alarm, State) ->
 acknowledge(AlarmId, Src, AckInfo,
             #al_state{ alarmlist = AList } = State) ->
     [{Key, Alarm}] = ets:lookup(AList, {AlarmId, Src}),
-    true = ets:insert(AList, {Key,Alarm#alarm{ ack_info = AckInfo,
-                                               state = acknowledged }}),
+    true = ets:insert(AList, {Key, Alarm#alarm{ ack_info = AckInfo,
+                                                state = acknowledged }}),
     {ok, State}.
 
 -spec unacknowledge(alarm_id(), alarm_src(), ack_info(), #al_state{}) ->
```

Comment:
The spec in line 79 uses a record, please define a type for the record and use that instead.